src/requirements/Points/PointsRequirement.tsx

```diff
@@ -0,0 +1,113 @@
+import Requirement, {
+  RequirementProps,
+  RequirementSkeleton,
+} from "components/[guild]/Requirements/components/Requirement"
+import { useRequirementContext } from "components/[guild]/Requirements/components/RequirementContext"
+import useGuild from "components/[guild]/hooks/useGuild"
+import Link from "components/common/Link"
+import Star from "static/icons/star.svg"
+
+const PointsRank = (props: RequirementProps): JSX.Element => {
+  const requirement = useRequirementContext()
+  const { guildPlatformId, guildId, minAmount, maxAmount } = requirement.data
+  const { name, urlName, guildPlatforms } = useGuild(guildId)
+  const { id: currentGuildId } = useGuild()
+
+  if (!guildPlatforms) return <RequirementSkeleton />
+
+  const pointsReward = guildPlatforms.find((gp) => gp.id === guildPlatformId)
+
+  return (
+    <Requirement
+      image={pointsReward.platformGuildData.imageUrl ?? <Star />}
+      {...props}
+    >
+      {minAmount
+        ? `Have a rank between ${minAmount} - ${maxAmount} on the `
+        : `Be in the top ${maxAmount} members on the `}
+      <Link
+        colorScheme={"blue"}
+        href={`/${urlName}/leaderboard/${pointsReward.id}`}
+      >{`${pointsReward.platformGuildData.name} leaderboard`}</Link>
+      {guildId !== currentGuildId && (
+        <>
+          {`in the `}
+          <Link colorScheme={"blue"} href={urlName}>
+            {name}
```

Comment:
We could add a skeleton loader here, since sometimes the loading time might be longer 👀